datadog-checks-base/datadog_checks/utils/subprocess_output.py

```diff
@@ -0,0 +1,35 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2018 Datadog, Inc.
+import logging
+
+try:
```

Comment:
I made a comment here on a slightly outdated PR, but this should still hold true. Please see comment below even though it's marked as stale.